github_actions/lib/dependabot/github_actions/file_parser.rb

```diff
@@ -55,7 +55,7 @@ def workfile_file_dependencies(file)
       def build_github_dependency(file, string)
         details = string.match(GITHUB_REPO_REFERENCE).named_captures
         name = "#{details.fetch('owner')}/#{details.fetch('repo')}"
-        url = "https://github.com/#{name}"
+        url = "https://#{source.hostname}/#{name}"
```

Comment:
I was wondering if we should also grab the scheme from the source, but looks like it isn't available, and people should really use ssl so 🤷